pom.xml

```
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>edge-patron</artifactId>
-  <version>4.7.0-SNAPSHOT</version>
+  <version>4.7.1-SNAPSHOT</version>
```

I believe this is a mainline feature release.

The next version for a mainline release should be the next minor or major (as described [here](https://dev.folio.org/guidelines/release-procedures/#prepare-and-perform-the-source-release)), not a patch version. As I understand it, patch versions should only be used in the long lived release branches.

```suggestion
  <version>4.8.0-SNAPSHOT</version>
```

---

updated